packages/src/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Internal dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+const {
+    registerBlockType,
+    unstable__bootstrapServerSideBlockDefinitions, // eslint-disable-line camelcase
+} = wp.blocks;
+
+/**
+ * Function to register core blocks provided by the block editor.
+ */
+
+const registerGlutenblocks = () => {
+    [
+    ].forEach((block) => {
+        if (!block) {
```

Comment:
How a block could be not defined?

Reply:
If the developer forgot to import it earlier.

Reply:
I've added this comment `// Empty for now since there is no blocks in the package` in the empty array to avoid confusion